redev_comm.h

```diff
@@ -151,40 +156,40 @@ class NoOpComm : public Communicator<T> {
 
 
 /**
- * The AdiosComm class implements the Communicator interface to support sending
+ * The AdiosPtnComm class implements the Communicator interface to support sending
  * messages between the clients and server via ADIOS2.  The BP4 and SST ADIOS2
  * engines are currently supported.
- * One AdiosComm object is required for each communication link direction.  For
+ * One AdiosPtnComm object is required for each communication link direction.  For
  * example, for a client and server to both send and receive messages one
- * AdiosComm for client->server messaging and another AdiosComm for
+ * AdiosPtnComm for client->server messaging and another AdiosPtnComm for
  * server->client messaging are needed. Redev::BidirectionalComm is a helper
  * class for this use case.
  */
 template<typename T>
-class AdiosComm : public Communicator<T> {
+class AdiosPtnComm : public Communicator<T> {
   public:
     /**
-     * Create an AdiosComm object.  Collective across sender and receiver ranks.
+     * Create an AdiosPtnComm object.  Collective across sender and receiver ranks.
      * Calls to the constructor from the sender and receiver ranks must be in
      * the same order (i.e., first creating the client-to-server object then the
      * server-to-client link).
      * @param[in] comm_ MPI communicator for sender ranks
      * @param[in] recvRanks_ number of ranks in the receivers MPI communicator
      * @param[in] eng_ ADIOS2 engine for writing on the sender side
      * @param[in] io_ ADIOS2 IO associated with eng_
-     * @param[in] name_ unique name among AdiosComm objects
+     * @param[in] name_ unique name among AdiosPtnComm objects
```

Comment:
Since we now have the partitioned and global communicators, specify that this communicator works on the partitioned data.